implementations/rust/ockam/ockam_identity/src/secure_channel/handshake/handshake_worker.rs

```diff
@@ -320,6 +321,7 @@ impl HandshakeWorker {
 
             WorkerBuilder::new(encryptor)
                 .with_mailboxes(Mailboxes::new(main_mailbox, vec![api_mailbox]))
+                .terminal(self.addresses.encryptor.clone())
```

Comment:
Should we also add `IdentityId` to the attributes?

Reply:
It would be nice, my reasoning so far was to keep `ockam_node` and `ockam_identity` decoupled (and in general avoiding identity-specific concepts within node), what's your take?